File: programming files/sum_even_fibonacci.cc
//Bugs introduced BS

int main()
{
    int a = 0, b = 1;
    int sum_even = 0;
    while (b < 4000000) {
        if (b % 2 == 0) {
            sum_even += b;
        }
        int temp = b;
        b = a + b;
        a = tem;
    }
    std::cout << sum_even << std::endl;


}

// answer should be 4613732

File: programming files/sum_even_fibonacci.java
//er
public class sum_even_fibonacci {
    public static void main(String[] args) {
        int a = 0, b = 1
        int sumEven = 0
        while (b < 4000000) {
            if (b % 2 == 0) {
                sumEven += a
            }
            temp = b
            b = a + b
            a += temp
        }
        System.out.prinln(sumEven)
    }
}

// answer should be 4613732
